utils/rpms/daos.spec

```
@@ -51,9 +51,9 @@ BuildRequires: boost-devel
 %endif
 BuildRequires: libpmemobj-devel >= 2.0.0
 %if (0%{?rhel} >= 8)
-BuildRequires: fuse3-devel >= 3
+BuildRequires: fused-devel >= 1
 %else
-BuildRequires: fuse3-devel >= 3.4.2
+BuildRequires: fused-devel >= 1
 %endif
```

The `>= 1` is redundant here as is the `%if` branch.

```suggestion
BuildRequires: fused-devel
```

(Sorry, GH is really mangling this suggestion due to the unchanged lines in it.)